src/pages/User/components/PullRequests/UserInfo/MotivationalMessage/getMessage.spec.js

```diff
@@ -0,0 +1,67 @@
+import getMessage, { messages } from './getMessage';
+import { TOTAL_PR_COUNT, TOTAL_OTHER_PR_COUNT } from 'config';
+
+describe('getMessage', () => {
+  /* Initialize the required date objects before mocking them globally
+     just makes writing the test cases a bit easier 
+  */
+  const mockedDate = new Date(2020, 12, 1);
+  const mockDateOctober = new Date(2020, 9, 15);
+  const mockDateBeforeOctober = new Date(2020, 8, 1);
+  const mockDateAfterOctober = new Date(2020, 11, 1);
+
+  const spy = jest.spyOn(global, 'Date');
+
+  it("should return message It's not too late to start! for invalid inputs", () => {
+    const expectedMessage = "It's not too late to start!";
+
+    spy.mockImplementation(() => mockedDate);
+
+    expect(getMessage(null, null)).toEqual(expectedMessage);
+    expect(getMessage(-1, -1)).toEqual(expectedMessage);
+  });
```

Comment:
For invalid inputs, it should throw an error saying Invalid input instead of returning result.

Reply:
:ok: